plugins/python-build/share/python-build/activepython-2.7.14

```diff
@@ -0,0 +1,13 @@
+case "$(activepython_architecture 2>/dev/null || true)" in
+"linux-x86_64" )
+  install_package "ActivePython-2.7.14.2717-linux-x86_64-glibc-2.12-404899" "http://downloads.activestate.com/ActivePython/releases/2.7.14.2717/ActivePython-2.7.14.2717-linux-x86_64-glibc-2.12-404899.tar.gz" "activepython" verify_py27
+  ;;
+* )
+  { echo
+    colorize 1 "ERROR"
+    echo ": The binary distribution of Active Python is not available for $(activepython_architecture 2>/dev/null || true)."
```

Comment:
they have distributions for macOS, x86 linux and windows as well

Reply:
I'm aware of the Windows and OSX distributions, however, I don't use either of those platforms and am not able to test them. 

X86 is included, only Python 3.6 includes an x86 package.